packages/fields-auto-increment/src/Implementation.js

```diff
@@ -40,10 +40,10 @@ export class AutoIncrementImplementation extends Implementation {
       ...this.inInputFields(this.gqlType),
     ];
   }
-  get gqlUpdateInputFields() {
+  gqlUpdateInputFields() {
```

Comment:
This function doesn't accept any param as mentioned in the changeset. Should it be: 
```js
 gqlUpdateInputFields({schemaName})
```
Same for `gqlCreateInputFields`.

Reply:
It accepts the parameter, it just happens to ignore them. We don't specify unused arguments in our method definitions, so what we have here is correct.

A future PR will make use of the `{ schemaName }` argument specifically in the `Relationship` field type.